app/controllers/accounts_controller.rb

```diff
@@ -19,6 +19,7 @@ def summary
   end
 
   def list
+    render layout: false
```

Comment:
Bug that has been here for a while—turbo frame was linking to itself since the sidebar (part of the layout) calls this.